api/functions/decideScore.js

```diff
@@ -1,13 +1,22 @@
-import logger from "../utils/logger.js";
-
 /**
  * Decides a score for a userbased on the number of messages and reactions
  * @param {number} messages - The number of messages
  * @param {number} reactions - The number of reactions
  * @returns {number} The score
  */
```

Comment:
It would be good to update this jsdoc comment explaining how and why we're calculating the score this way